spec/models/application_dates_spec.rb

```diff
@@ -14,22 +14,34 @@
   end
 
   describe '#submitted_at' do
-    context 'when not continuous applications', continuous_applications: false do
+    context 'when not continuous applications' do
       it 'returns date that application was submitted on' do
         expect(application_dates.submitted_at).to eql(submitted_at)
       end
     end
```

Comment:
This **is** continuous applications now. The feature flag is on by default and the recruitment cycle year is > 2023.

(But submitted_at is 2019)

Reply:
Done. I removed this context because there is the same duplicate under the same spec.